components/cluster-secret-store/base/appsre-stonesoup-vault-secret-store.yaml

```diff
@@ -52,3 +52,4 @@ spec:
         - integration-service
         - ci-helper-app
         - mintmaker
+        - tracing
```

Comment:
the namespace is `konflux-otel `, not `tracing`.
```suggestion
        - konflux-otel 
```

Reply:
TYVM, fixing